### ISSUE TITLE
As a Gitcoin admin, there is an over usage of override statuses, which lead to incorrect statuses that leave funders confused about the true status of their bounties

### ISSUE THREAD

**Issue by frankchen07** [gitcoinco/web#3240]
### User Story

As a Gitcoin admin, there is an over usage of override statuses, which lead to incorrect statuses that leave funders confused about the true status of their bounties. There have been an increasing number of support requests from people whose bounties have an override status on them.

Example: https://github.com/gitcoinco/web/issues/2764
Example (fixed): https://github.com/prysmaticlabs/prysm/issues/543

### Current Behavior

More or less careless usage of override statuses, issue explorer checkin ambassadors have been notified and provided a sheet of current bounties with override statuses.

### Possible Behaviors

1. more responsible usage of override statuses
2. making the rounds to check issues that have override statuses and change them back as necessary
3. expiration of override statuses
4. remove the ability to override statuses

**Comment by mbeacom** [comment]
I voiced concerns about introducing the override mechanism from the jump, so going back to those... I think we should definitely eliminate this option all together and focus on making the status updates more meaningful and accurate.

**Comment by kuhnchris** [comment]
I agree with @mbeacom - we should fix the source of the problem, not just use a work-around.
Can we check to see who is using this function mostly? if so, ask the core member what the issue is and let's try to work from there to improve the process to get a clean solution going.

Also, as an intermediate solution, only allow bounty status overrides with a reference to a issue here on github? at least then you can see a obvious reason for the override. But that's a process topic.

If bounties status cause so many troubles to begin with, we could think about increasing the logging for those to try to follow up in a 2nd step.

-Chris

**Comment by frankchen07** [comment]
I'll add some market context here:

I've used the override bounty status for:

- marking issues as done when people just tip out and leave the bounty hanging 
- marking issues as cancelled if the issue had expired and the funder explicitly wanted it cancelled.
- marking issues as open when an abandoned contributor was taking up the issue (should be remove the contributor via admin)
- marking issues as open when incorrect or empty PRs were submitted

I realized that override statuses are "hard" overrides, meaning they stay static. I was under the impression that they were non-static, just a forcible move to a status that could change later. So I was definitely part of the problem - luckily the number of issues I did this for was less than 20ish, and I wised up soon after (and reversed a couple of them).

I've posted in the issue explorer checkin channel (and probably should do so for the ambassadors as well), a CSV of all bounties that currently have a override status so we at least have a starting point on which issues need to be doubled checked on, as a very first step to remedying the situation. Agree that this is only a short-term solution, and moving towards a long-term solution is ideal.

From the context of an admin though, I was using override statuses to "fix" incorrect statuses, but that was not the intention of override statuses in the first place. For the use cases where we want statuses to change, it seems we do have a non-override option for them in the first place - (admin, changing the idx_status), or:

1. expired bounty -> open bounty (we remarket)
2. started bounty -> open bounty (we remove the interested party from admin)
3. submitted bounty -> open bounty (admin)
4. bounty -> cancelled bounty (admin)

Perhaps people have been using the override status because it's readily accessible up top, and simply easy to reach, rather than going into admin and figuring out which fields to modify for the correct scenario.
